test/e2e/lib/reporter/magellan-reporter.js

```diff
@@ -14,24 +12,11 @@ const Reporter = function () {};
 
 util.inherits( Reporter, BaseReporter );
 
-const log = ( testRun ) =>
-	new Writable( {
-		write( chunk, encoding, callback ) {
-			const msg = chunk.toString( 'utf8' );
-			msg.split( '\n' ).forEach( ( line ) => {
-				if ( line.length ) {
-					logger.log( '--> ' + testRun.guid + ' ' + line );
-				}
-			} );
-			callback( null );
-		},
-	} );
-
 Reporter.prototype.listenTo = function ( testRun, test, source ) {
 	// Print STDOUT/ERR to the screen for extra debugging
 	if ( process.env.MAGELLANDEBUG ) {
-		source.stdout.pipe( log( testRun ) );
-		source.stderr.pipe( log( testRun ) );
+		source.stdout.pipe( process.stdout );
```

Comment:
The new reporter `mocha-teamcity-reporter` works by outputting TeamCity "commands" to the console. This code was prefixing those commands with the worker info from Magellan, but those extra bits make TeamCity unhappy.